diff -rupN roundup-2.6.0.orig/roundup/cgi/client.py roundup-2.6.0/roundup/cgi/client.py
--- roundup-2.6.0.orig/roundup/cgi/client.py	2026-06-03 02:23:09.000000000 +0000
+++ roundup-2.6.0/roundup/cgi/client.py	2026-09-13 17:29:50.784681702 +0000
@@ -2704,6 +2704,8 @@ class Client:
         """
         try:
             call(*args, **kwargs)
+        except socket.timeout:
+            pass
         except socket.error as err:
             err_errno = getattr(err, 'errno', None)
             if err_errno is None:
